### PR TITLE
*/*: Bump qalculate packages

### PR DESCRIPTION
**Pull request by csfore**
- sci-calculators/qalculate-{gtk,qt} to 5.0.0
- sci-libs/libqalculate to 5.0.0
- SONAME did not change in libqalculate

All tests pass on amd64.

### CODE REVIEW

File: sci-calculators/qalculate-gtk/qalculate-gtk-5.0.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+inherit xdg
+
+DESCRIPTION="Modern multi-purpose calculator"
+HOMEPAGE="https://qalculate.github.io/"
+SRC_URI="https://github.com/Qalculate/${PN}/releases/download/v${PV}/${P}.tar.gz"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~alpha amd64 ~arm64 ~hppa ~ppc ~ppc64 ~sparc ~x86 ~amd64-linux ~x86-linux"
```
Comment (thesamesam) [inline sci-calculators/qalculate-gtk/qalculate-gtk-5.0.0.ebuild:16]: pkgcheck scan --commits

**Review by thesamesam — CHANGES_REQUESTED** [review]
One commit per package please

File: sci-calculators/qalculate-gtk/qalculate-gtk-5.0.0.ebuild (hunk 1 above)
Comment (csfore) [reply sci-calculators/qalculate-gtk/qalculate-gtk-5.0.0.ebuild]: Fixed!

**Comment by AndrewAmmerlaan** [comment]
Thanks :+1:

**Pull request closed without merging**